crates/extensions/c8y_mapper_ext/src/inventory.rs

```diff
@@ -4,58 +4,23 @@ use crate::error::ConversionError;
 use crate::fragments::C8yAgentFragment;
 use serde_json::json;
 use serde_json::Value as JsonValue;
-use std::fs::File;
-use std::io::Read;
-use std::path::Path;
 use tedge_api::entity::EntityType;
 use tedge_api::entity_store::EntityTwinMessage;
-use tedge_api::mqtt_topics::Channel;
 use tedge_api::mqtt_topics::EntityTopicId;
 use tedge_config::models::TopicPrefix;
 use tedge_mqtt_ext::MqttMessage;
 use tedge_mqtt_ext::Topic;
-use tracing::info;
 
-const INVENTORY_FRAGMENTS_FILE_LOCATION: &str = "device/inventory.json";
 const INVENTORY_MANAGED_OBJECTS_TOPIC: &str = "inventory/managedObjects/update";
 
 impl CumulocityConverter {
     /// Creates the inventory twin messages with fragments from inventory.json file
     pub(crate) fn base_inventory_twin_data(&mut self) -> Result<Vec<MqttMessage>, ConversionError> {
```

Comment:
The comment and the function name are misleading now since it doesn't publish to `twin` topic.